### DISCUSSION TITLE
MeshCentral got locked

### DISCUSSION THREAD

**Discussion by lyyiheang** [Ylianst/MeshCentral#5152] [Q&A]
🙏🏻

**Comment by SomeGuru** [comment]
Looks to be a browser overlord here, so question in my mind is if they are blocking javascript calls to the browser in some fashion. I would almost bet that they are, and part of that would be the screen sharing part of JavasScript that could be blocked by the security mechanism in Respondus LockDown Browser. I would check settings and see if you can make the domain (i.e., mesh.school.k-12.st.edu) on approved list of browsers. Also the application itself... I don't have access to tru this application to give better details but I would assume there is a way to legitify the display mechanism through the controls. Keep us posted what you find if you would please...

-SomeGuru

**Reply by SomeGuru** [reply]
Correct, I think the security part is kicking in and blocking the display javascript for good security reasons... it could be seen as a workaround or bypass by the security app side of it. So therefore the KM functions are not hindered, just the video part. 

-SomeGuru

**Reply by lling345** [reply]
Hi @SomeGuru. Do you have anything specific in mind on how to deal with this. This could be the case with any popup window in the future.

With regard to your comment:
"check settings and see if you can make the domain (i.e., mesh.school.k-12.st.edu) on approved list of browsers."
Where do we do this?

**Reply by SomeGuru** [reply]
In the settings, it should have an option for accepting running scripts and other related for the browser. Usually all browsers allow you to approve list the domains you trust. Also could be listed as trust.

-SomeGuru

**Comment by NiceGuyIT** [comment]
According to [LockDown Browser overview][], screen capture is disabled. MeshCentral server or agent can't override this. You'll need to contact their support to see if it's possible to disable that feature for a specific application (meshagent).
> Printing and screen capture functions are disabled

This same thing happens with [KeePassXC's Screenshot Security][] feature which can be disabled with the `--allow-screencapture` flag.

[LockDown Browser overview]: https://web.respondus.com/he/lockdownbrowser/
[KeePassXC's Screenshot Security]: https://keepassxc.org/docs/KeePassXC_GettingStarted#_screenshot_security

**Reply by lling345** [reply]
Hi. Were you able to find a solution to this?

**Reply by silversword411** [reply]
You're trying to use an app (MeshCentral) for remote control...that works in a web browser....and you're trying to get it to work on a web browser (lockdown browser) that's explicitly designed and engineered to stop remote control functions from working.

Really?

**Reply by nbviet300689** [reply]
![20240327_114030](https://github.com/Ylianst/MeshCentral/assets/151039215/41eec549-15b5-4a3a-aa5d-9c8cbe0f7449)
Has anyone found a bypass solution with Meshcentral?
